src/VS.Web.CG.Mvc/Templates/Identity/Bootstrap3/Pages/Account/Manage/Account.Manage.ExternalLogins.cs.cshtml

```diff
@@ -105,10 +115,11 @@ namespace @thisNamespace
                 return NotFound($"Unable to load user with ID '{_userManager.GetUserId(User)}'.");
             }
 
-            var info = await _signInManager.GetExternalLoginInfoAsync(await _userManager.GetUserIdAsync(user));
+            var userId = await _userManager.GetUserIdAsync(user);
```

Comment:
you don't need to call GetUserId in the scaffolded code, user.Id works and is cleaner